src/status_im/utils/datetime.cljs

```diff
@@ -111,6 +111,12 @@
 (defn now-ms []
   (to-long (now)))
 
+(defn now-s! []
```

Comment:
what the purpose of `!` ?

Reply:
side effect in this case, though we don't have a convention

Reply:
Since now-ms doesn't have a bang I wouldn't put one here

Reply:
ok